File: Java/controletarefas/src/main/java/br/com/iteris/universidade/controletarefas/domain/entity/Tarefa.java
package br.com.iteris.universidade.controletarefas.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Tarefa {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int idTarefa;
    private String titulo;
    private String descricao;
    private boolean concluido;
    private int prioridade;
}

File: C#/ControleTarefas/ControleTarefas/Domain/Entity/Tarefa.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace ControleTarefas.Domain.Entity
{
	[Table("Tarefas")]
    public class Tarefa
    {

		[Key]
		public int Id { get; set; }
		[Required]
		[StringLength(255)]
		public string Titulo { get; set; }
		[StringLength(255)]
		public string Descricao { get; set; }
		[Required]
		public bool Concluido { get; set; }

		[Required]
		public int Prioridade { get; set; }

	}
}
